CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+
+* Added `httpx.SSLContext` class and `ssl_context` argument. (#3022)
+* Removed `cert` and `verify` arguments, you should use the `ssl_context=...` instead. (#3022)
```

Comment:
* The `verify` and `cert` arguments continue to function, but are deprecated and will raise warnings. Use the `ssl_context = httpx.SSLContext()` API instead. (#3022, #3335)